### PR TITLE
37c3 potluckctf - tacocat writeup

### PR DESCRIPTION
**Pull request by Stefan-Radu**
cool python escape wooo

### CODE REVIEW

File: content/37c3_potluckctf_2023/_index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: 37c3 Potluckctf 2023
+date: 2023-12-29
+description: Writeups for [37c3 Potluckctf 2023]
+
```
Comment (Mathis-Dory) [inline content/37c3_potluckctf_2023/_index.md:5]: No final place ?

File: content/37c3_potluckctf_2023/tacocat.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+This can give us the flag, or a shell, but the catch is that:
+* the input must be a palindrome
+* we cannot use comments
+* sum of ASCII values for each character in the input has to be equal to 69 (modulo 256)
+
+I had multiple ideas, including using string formatting to insert comments,
+using tuples which are evaluated element by element and to use triple quotes
+strings as a form of comment. Sadly I had no success with any of them, but I'm
+happy to see that some of my ideas were in a solution by `pspaul @
+[FluxFingers](https://fluxfingers.net/)`, which I we will discuss below.
```
Comment (Mathis-Dory) [inline content/37c3_potluckctf_2023/tacocat.md:43]: The link is not clickable

```suggestion
happy to see that some of my ideas were in a solution by `pspaul` @
[FluxFingers](https://fluxfingers.net/), which I we will discuss below.
```

**Review by Mathis-Dory — CHANGES_REQUESTED** [review]

**Comment by Costinteo** [comment]
please @Mathis-Dory don't merge main into branches, do a rebase instead. Otherwise we'll end up with lots of useless commits on the main commit history

**Comment by Mathis-Dory** [comment]
> please @Mathis-Dory don't merge main into branches, do a rebase instead. Otherwise we'll end up with lots of useless commits on the main commit history

WDYM ? the branch was outdated and Github only gave the option to merge

File: content/37c3_potluckctf_2023/_index.md (hunk 1 above)
Comment (Stefan-Radu) [reply content/37c3_potluckctf_2023/_index.md:5]: no, I actually don't know =)) I just joined some guys playing. I don't even know the team name

Comment (Mathis-Dory) [reply content/37c3_potluckctf_2023/_index.md:5]: ookey

**Review by Mathis-Dory — APPROVED** [review]

**Pull request merged**